nmap.go

```diff
@@ -1149,6 +1149,20 @@ func WithBadSum() func(*Scanner) {
 
 /*** Output ***/
 
+// WithVerbosity sets and increases the verbosity level of nmap.
+func WithVerbosity(level int) func(*Scanner) {
```

Comment:
WDYT of making sure that the level is within allowed boundaries? (1 to 6 if I recall correctly)

Reply:
I think you are correct. Let me verify the levels of debugging and verbosity and see if I can put some logic in there